charts/cluster-addons/README.md

```diff
@@ -269,6 +269,14 @@ By default, Grafana is only available from within the cluster and must be access
 kubectl -n monitoring-system port-forward svc/kube-prometheus-stack-grafana 3000:80
 ```
 
+## Intel Device Plugin
+
+Deploys in the Intel Device Plugin operator for managing Intel devices,
+alongside GPU plugin CRDs for the operator to add Intel GPU resources to nodes. [Upstream GPU options](https://github.com/intel/helm-charts/tree/main/charts/gpu-device-plugin#configuration) can be modified by overriding the defaults in
```

Comment:
Nit: line length

Reply:
FWIW, I think we should add a linter to catch this. I try to ignore that.